scripts/build_docker.sh

```diff
@@ -19,7 +19,7 @@ docker buildx build \
     ${LOAD_OR_PUSH} \
     --platform=${PLATFORM} \
     ${OLLAMA_COMMON_BUILD_ARGS} \
-    -f Dockerfile \
+    -f Dockerfile${NEW_MAKEFILE} \
```

Comment:
Not to over-iterate on this but is it possible to make this, and the flags in the builds script the same? i.e. `OLLAMA_NEW_RUNNERS=1`

Reply:
I'll adjust this and move the new Dockerfile under the llama directory for now.

To build the new runners with the various helper scripts you do
```
OLLAMA_NEW_RUNNERS=1 ./scripts/build_linux.sh

OLLAMA_NEW_RUNNERS=1 ./scripts/build_docker.sh

OLLAMA_NEW_RUNNERS=1 ./scripts/build_darwin.sh

$env:OLLAMA_NEW_RUNNERS="1"
./scripts/build_windows.ps1
```